community.md

```diff
@@ -7,12 +7,12 @@ title: Community
 
 ### The xmonad community
 
-*   [the irc channel](http://haskell.org/haskellwiki/IRC_channel): `#xmonad@chat.freenode.org` (join it via [webchat](https://webchat.freenode.net/))
+*   [the irc channel](https://wiki.haskell.org/IRC_channel): `#xmonad@chat.freenode.org` (join it via [webchat](https://webchat.freenode.net/))
 *   [the matrix channel](https://matrix.to/#/#freenode_#xmonad:matrix.org): this is linked to the above IRC channel
-*   [xmonad on twitter](http://twitter.com/xmonad)
+*   [xmonad on twitter](https://twitter.com/xmonad)
 *   [the subreddit](https://old.reddit.com/r/xmonad/)
-*   [the mailing list](https://mail.haskell.org/cgi-bin/mailman/listinfo/xmonad) (archives: [pipermail](http://www.haskell.org/pipermail/xmonad/), [gmane](http://dir.gmane.org/gmane.comp.lang.haskell.xmonad))
-*   [the wiki](http://haskell.org/haskellwiki/Xmonad)
+*   [the mailing list](https://mail.haskell.org/cgi-bin/mailman/listinfo/xmonad) (archives: [pipermail](https://mail.haskell.org/pipermail/xmonad/), [gmane](http://dir.gmane.org/gmane.comp.lang.haskell.xmonad))
```

Comment:
I do think that gmane should just be removed as well; I doubt it'll be coming back.